src/win/udp.c

```diff
@@ -924,5 +924,44 @@ int uv__udp_try_send(uv_udp_t* handle,
                      unsigned int nbufs,
                      const struct sockaddr* addr,
                      unsigned int addrlen) {
-  return UV_ENOSYS;
+
+  DWORD bytes;
+  const struct sockaddr* bind_addr;
+  int err;
+
+  assert(nbufs > 0);
+
+  /* already sending a message */
+  if (handle->send_queue_count != 0)
+  return UV_EAGAIN;
+
+  if (!(handle->flags & UV_HANDLE_BOUND)) {
+    if (addrlen == sizeof(uv_addr_ip4_any_)) {
+      bind_addr = (const struct sockaddr*) &uv_addr_ip4_any_;
+    }
+    else if (addrlen == sizeof(uv_addr_ip6_any_)) {
```

Comment:
This should go on the same line as the previous curly brace. But, house style is to leave off the curly braces if all of the branches are a single statement.